src/TestExplorer/TestRunner.ts

```diff
@@ -665,6 +684,17 @@ export class TestRunner {
             return;
         }
 
+        if (isRelease(this.testKind)) {
+            buildAllTask.definition.args = [
```

Comment:
question: if you then run the `Tasks: Run Task` command and select build all from the `swift` folder, does this have the release args since you're caching build all task?

Reply:
Verified this doesn't cache the changes here on the task used w/ Tasks: Run Task > Build All